.github/workflows/add-pr-approval-label.yml

```diff
@@ -17,7 +17,6 @@ jobs:
       - uses: ./
         with:
           helper: add-pr-approval-label
-          teams: Repo-Maintainers
           login: ${{ github.event.review.user.login }}
```

Comment:
Should we not support both teams and CODEOWNERS file?

Reply:
We do, I am just omitting `teams` for this workflow